app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+  belongs_to :organisation
+  has_many :contact_list_users
+  has_many :contact_lists, through: :contact_list_users
+  has_many :users, through: :contact_lists
```

Comment:
Does a user have many users?  I would have thought if there was a relationship here it would have been the list of contacts from all the lists the user is assigned to

Reply:
My rails is terrible, but something like:
`has_many :contacts, through: :contact_lists`

Reply:
Absolutely my bad, of course this is a mistake, fixing now :)